templates/master.yaml.tmpl

```diff
@@ -1676,6 +1706,7 @@ storage:
             labels:
               k8s-app: nginx-ingress-controller
           spec:
+            externalTrafficPolicy: Local
```

Comment:
This one is tricky a bit. For services of type `NodePort` or `LoadBalancer` filtering works only with `Local` external traffic policy. In this mode source IP is not changed. That in general affects traffic balancing as requires  nginx ic on every worker. But in case of control-plane that is the case - every worker has IC pod. So no harmful impact here. Onprem we don't need all of this as customers anyway have network access to control-plane API.

Just bringing this for awareness of @giantswarm/sig-infrastructure